env.sample

```diff
@@ -17,6 +17,9 @@ DISCOVERY_USERNAME=03c25743-4728-448e-b3ed-3b198e6edd65
 DISCOVERY_PASSWORD=4LDhpX27q7QZ
 DISCOVERY_ENVIRONMENT_ID=a2e0e223-aec9-458e-95b5-d4d618e1ae6d
 DISCOVERY_COLLECTION_ID=d06e9ddf-7c27-4dce-b80f-dca8768a72d8
+# IBM Logo Store min score value = .15
```

Comment:
Wondering if we need even more comments, but in the current context I think this is OK.